flink-formats/flink-sql-parquet/pom.xml

```diff
@@ -42,12 +42,14 @@ under the License.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-parquet</artifactId>
 			<version>${project.version}</version>
+			<optional>${flink.markBundledAsOptional}</optional>
 		</dependency>
 
 		<dependency>
 			<groupId>org.apache.parquet</groupId>
 			<artifactId>parquet-avro</artifactId>
 			<version>${flink.format.parquet.version}</version>
+			<optional>${flink.markBundledAsOptional}</optional>
```

Comment:
do we really need this? this dependency is not shaded... :thinking:

Reply:
according to the shade-plugin configuration it is.
```
<include>org.apache.parquet:parquet-avro</include>
```

Reply:
true, I must have mixed up things. :+1: